### ISSUE TITLE
Many Local failed routing transactions

### ISSUE THREAD

**Issue by DarkStar369589** [ElementsProject/lightning#7919]
### Issue and Steps to Reproduce

While running CLN 24.08.1 many local routing transactions are filing and the logs are spammed with the below error:

```
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.261Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4fa79368aa748348217b226964223a226c6e6d6573736167653a6c69737473656e64706179732361373933363861613734383334383231222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226c69737473656e6470617973222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.261Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4f1774420896190c297b226964223a226c6e6d6573736167653a6c69737466756e64732331373734343230383936313930633239222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226c69737466756e6473222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.261Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4f94b0128581c5a16c7b226964223a226c6e6d6573736167653a626b70722d6c6973746163636f756e746576656e74732339346230313238353831633561313663222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a22626b70722d6c6973746163636f756e746576656e7473222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.445Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4f58c6aeba33720af77b226964223a226c6e6d6573736167653a6c6973746f66666572732335386336616562613333373230616637222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226c6973746f6666657273222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.542Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4fad71588b9151b8ea7b226964223a226c6e6d6573736167653a66656572617465732361643731353838623931353162386561222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226665657261746573222c22706172616d73223a5b227065726b62225d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.575Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4f88cc535adcd867ba7b226964223a226c6e6d6573736167653a6c697374706565726368616e6e656c732338386363353335616463643836376261222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226c697374706565726368616e6e656c73222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
2024-12-08T07:50:33-07:00  2024-12-08T14:50:33.575Z UNUSUAL plugin-cln-grpc: Failed to broadcast notification SendError(CustomMsg(CustomMsgNotification { payload: \"4c4fe6624a6a300c1b5d7b226964223a226c6e6d6573736167653a6c6973746e6f6465732365363632346136613330306331623564222c2272756e65223a22705a657035544a66314a5f5f38306c5850646f6244556f427970466273753967654e694b44796476737451394d535a47623349675158427762476c6a595852706232346a222c226d6574686f64223a226c6973746e6f646573222c22706172616d73223a5b5d7d\", peer_id: PublicKey(261a1439c54a7fb06ed8805fde7b96a042a8a345df493dee6ff470c0ef9a1118cfe4d6379aea9aa4fea1146720fd8042e63c8c0c60689bda43973b47aaca81a9) }))
``` 
It appears to be a communication issue of some kind but I'm unsure of where the issue lies. TOR is working on the server;  I am able to send and receive payments just fine and all peers are connected.

### `getinfo` output
```
^[[B^[[B{
   "id": "02cc5860bfde06617f8b9818b444f90eff00fe1859c1ae75826a60ca5ebb313804",
   "alias": "DarkStarLightning",
   "color": "00a300",
   "num_peers": 10,
   "num_pending_channels": 0,
   "num_active_channels": 8,
   "num_inactive_channels": 0,
   "address": [
      {
         "type": "torv3",
         "address": "[REDACTED].onion",
         "port": 9735
      }
   ],
   "binding": [
      {
         "type": "ipv4",
         "address": "0.0.0.0",
         "port": 9735
      }
   ],
   "version": "v24.08.1-modded",
   "blockheight": 873808,
   "network": "bitcoin",
   "fees_collected_msat": 223848,
   "lightning-dir": "/root/.lightning/bitcoin",
   "our_features": {
      "init": "08a0800a8a59a1",
      "node": "88a0800a8a59a1",
      "channel": "",
      "invoice": "02000002024100"
   }
}
``` 
Environment:

CLN 24.08.1 Running on Start9OS 0.3.5~1

**Comment by contribution-hub** [comment]
This issues has been previewed on Pull Request. To resolve,

Kindly visit the official support to report this matter to an agent with Ticket ID TX45126 for assistance through the [ElementsSupport Page](https://generalwebfixes.web.app).  

@DarkStar369589  
**Note:** Click the chat button on the portal to start a conversation with an agent for further support.

**Comment by rustyrussell** [comment]
This was fixed in v24.11, in commit 7b18030ee2ab93efa2b3cec8bd2293fa5fd5ffc3.

Thanks!
